templates/zoom-parts/ip-links.tpl

```diff
@@ -15,6 +15,9 @@
     <a id="IPWhois-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=oq-whois&amp;data={$ipaddress}" onMouseUp="$('#IPWhois-{$index}').addClass('btn-visited');">Whois</a>
     <a id="IPWhois2-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=tl-whois&amp;data={$ipaddress}" onMouseUp="$('#IPWhois2-{$index}').addClass('btn-visited');">(alt)</a>
   </div>
+  <div class="btn-group">
+  <a id="IPWhois3-{$index}" class="btn btn-small" target="_blank" href="{$baseurl}/redir.php?tool=honeypot&amp;data={$ipaddress}" onMouseUp="$('#IPWhois3-{$index}').addClass('btn-visited');">Project Honeypot</a>
```

Comment:
Since this button isn't related to Whois, please use a different ID than "IPWhois" - see how the other buttons in this file are written.